core/primitives/src/trie_key.rs

```diff
@@ -57,11 +57,14 @@ pub mod col {
     /// (`primitives::receipt::Receipt`).
     pub const BUFFERED_RECEIPT: u8 = 14;
     pub const BANDWIDTH_SCHEDULER_STATE: u8 = 15;
+    /// Stores `ReceiptGroupsQueueData` for the receipt groups queue
+    /// which corresponds to the buffered receipts to `receiver_shard`.
+    pub const OUTGOING_RECEIPT_GROUPS_QUEUE_DATA: u8 = 16;
+    /// A single item of `ReceiptGroupsQueue`. Values are of type `ReceiptGroup`.
+    pub const OUTGOING_RECEIPT_GROUPS_QUEUE_ITEM: u8 = 17;
```

Comment:
In general I agree that outgoing would be better, but now that we already have BUFFERED I'd rather keep things consistent. Yeah, it's just a nit, I'm not too attached to either name.

Reply:
I don't really like `INDICES`, it lies about what the data really is. There is a comment pointing to `ReceiptGroupsQueueData`, if someone is interested in what exactly the data is, they can quickly go there and take a look.

I'll change it to `BUFFERED_RECEIPT_GROUPS_QUEUE_DATA/ITEM`. I think Wac liked it as well, so that's 2 votes for this version.